_bibliography/marks-of-excellence.md

```diff
@@ -24,8 +24,8 @@ summary: 'Marks of Excellence (first published in 1997) offers a rigorous explor
   expanded edition will include at least 500 new images and 80 pages of new material,
   bringing this successful title right up to date.'
 mandatory: false
-site: https://fr.phaidon.com/store/design/marks-of-excellence-9780714864747
-isbn: 9780714864747
+site: https://www.fnac.com/a897063/MOLLERUP-P-Marks-of-excellence
```

Comment:
Editeur ?

Reply:
Par exemple sur Phaidon l'article n'est plus disponible https://www.phaidon.com/store/design/marks-of-excellence-the-history-and-taxonomy-of-trademarks-9780714864747/
Si je comprend bien, qu'importe l'éditeur, il vaut mieux proposer un lien vers l'éditeur plutôt qu'une marketplace Amazon/Fnac
Je peux chercher des liens vers les éditeurs dans ce cas